handler.js

```diff
@@ -15,46 +15,120 @@ const connection = mysql.createConnection({
   host: process.env.DB_HOST,
   user: process.env.DB_USER,
   password: process.env.DB_PASSWORD,
-  database: "grokit"
+  database: "grokit",
 });
 
 // get projects
 
 app.get("/projects", function (req, res) {
```

Comment:
It looks like when you merged master into your new branch you kept the old version of app.get and did not accept the changes coming from master. Is this right? In which case can you update the app.get endpoint in your branch to match the version in master here? https://github.com/GrokItTeam/grokit-backend/blob/master/handler.js